src/herald/start_user_internet_available.py

```diff
@@ -0,0 +1,48 @@
+# start_internet_available_user.py
+#
+# Copyright (C) 2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Main program for start-user-internet-available script.
+
+
+import os
+
+from herald.utils import get_logged_in_users, start_user_target
+
+
+def start_internet_available_user_target(user):
+    """Start the internet-available-user.target for a given user."""
+
+    return start_user_target('internet-available-user.target', user)
+
+
+def start_internet_available_user_target_for_all_users():
+    """
+    Start the internet-available-user.target for all currently
+    logged in users.
+    """
+
+    users = get_logged_in_users()
```

Comment:
This list isn't necessarily unique users, we might be wasting time doing this for the same user multiple times.

Reply:
Why isn't it unique? The function returns a set.

Reply:
Ah, I missed that bit. In which case, all's good.